Source/Classes/Core.js

```diff
@@ -130,15 +132,15 @@
 								Logger.info(data);
 							});**/
 							
-							if (os.platform() === 'win32') {
+							if (os2.platform() === 'win32') {
 								const ls = Process.spawn('D:\\Software\\Steam\\Steam.exe', [ '-applaunch', 322330, '-window' ], {
 									cwd: 'D:\\Software\\Steam\\',
 									detached: false
 								});
 							}
-							if (os.platform() === 'linux') {
-								const ls = Process.spawn(os.homedir() + '/.steam/steam/steamapps/common/Don''t Starve Together/bin/dontstarve', [ ], {
-									cwd: os.homedir() + '/.steam/steam/steamapps/common/Don''t Starve Together/bin/dontstarve',
+							if (os2.platform() === 'linux') {
+								const ls = Process.spawn('steam', [ 'steam://rungameid/322330'], {
```

Comment:
Wrong call; We wan't to debug the output of the executable from `Steam`. The `steam://`-Protocol won't start the game directly, Steam run's the game from the background-process.